tests/scripts/run-test-suites.pl

```diff
@@ -26,8 +26,8 @@
 
 use Getopt::Long;
 
-my $verbose;
-GetOptions( "verbose|v" => \$verbose );
+my $verbose = 0;
+GetOptions( "verbose|v+" => \$verbose );
```

Comment:
It is arguable, but `=i` seems more natural to me than `+`...

Reply:
I'd be happy to hear another recommendation here. I can go with whatever, so long as GetOpt does it well and the usability doesn't seem too bad.

Reply:
That sort of works, but it doesn't allow `./run-test-suites.pl -v` anymore, as that's interpreted as "verbose = 0". I think that's a major usability issue.

Reply:
I've discovered a new option in Getopt that let's me implement your number proposal easily.

```
: number [ desttype ]

Like :i , but if the value is omitted, the number will be assigned.
```

This let's me make the default value for a single `-v` be 1, instead of 0. I've amended the last commit to add this.

Reply:
:( `-vv` is the usual way to do this. It took me a while to realize that I needed to write `-v 2` and not `-vv` or `-v2`. Furthermore `:` in `Getopt::Long` has a quirky optional-argument syntax that prevents bunching short options or bunching the option's argument with the option, which is how optional arguments work with other command line parsers. I wish this had stayed `v+`.